packages/form-integrations/test/helpers/umbrella-form.js

```diff
@@ -64,11 +64,11 @@ export class UmbrellaForm extends LitElement {
           <lion-input-email name="email" label="Email"></lion-input-email>
           <lion-checkbox-group
             label="What do you like?"
-            name="checkers[]"
```

Comment:
Do we no longer strongly encourage the `[]` here for backend form integration?

Reply:
a group should never had it 😅  
all it's children should have gotten it...

and when using it we run into a different bug 🙈
but that is for another merge request

Reply:
It currently breaks if you do.
This could be fixed by changin line 88 of FormRegistrarMixin: `if (name.substr(-2) === '[]' && this._repropagationRole !== 'choice-group') {`